src/components/common/forms/form-field/form-field.js

```diff
@@ -0,0 +1,34 @@
+import WebComponent, { Component} from '#WebComponent';
+import css from './form-field.css?inline';
+
+
+export default Component ({
+    tagName: 'form-field',
+    styleCSS: css
+},
+
+class FormField extends WebComponent {
+    get value() {
+        return this._getDOM().querySelector('input').value;
+    }
+
+    render() {
+        const error = this.getAttribute('error') || '';
+        const rl = this.getAttribute('rl') || 'input';
+        const text = this.getAttribute('text') || 'text';
+        const type = this.getAttribute('type') || 'type';
```

Comment:
```suggestion
        const type = this.getAttribute('type') || 'text';
```

Este valor lo enchufamos en el atributo type de un input. Si miras la documentación, https://www.w3schools.com/html/html_form_input_types.asp, no se le puede pasar "type" como tal, por defecto si no le pasamos nada es "text", así podríamos respetarlo.